.github/actions/issues-nudge.js

```diff
@@ -0,0 +1,72 @@
+const { Octokit } = require("octokit");
+
+const token = process.env.GITHUB_TOKEN;
+const [owner, repo] = process.env.GITHUB_REPOSITORY.split("/");
+
+const octokit = new Octokit({ auth: token });
+
+async function check_issue(issue) {
+  const events = await octokit.paginate(octokit.rest.issues.listEvents, {
+    owner: owner,
+    repo: repo,
+    issue_number: issue.number,
+    per_page: 100,
+  });
+  const added_to_project = events.filter(e => e.event == "added_to_project");
+  const removed_from_project = events.filter(e => e.event == "removed_from_project");
+  const projects = added_to_project.length - removed_from_project.length;
```

Comment:
Isn't there a way to get an issue's projects, rather than iterating through all its events?

Reply:
we could not find one

Reply:
Not that I could spot any in GitHub's API reference. The milestone comes for free, but that's probably because there's only a single one. We could go the reverse way though, enumerating all issues in all projects and then figuring out which ones are missing...

Reply:
Now there with GraphQL